interceptors/logging/interceptors.go

```diff
@@ -50,64 +52,119 @@ func (c *reporter) PostCall(err error, duration time.Duration) {
 	logger.With(c.opts.durationFieldFunc(duration)...).Log(c.opts.levelFunc(code), fmt.Sprintf("finished %s %s call", c.kind, c.typ))
 }
 
-func (c *reporter) PostMsgSend(interface{}, error, time.Duration) {}
+// PostMsgSend logs the details of the servingObject that is flowing out of the rpc.
+// resp object wrt server.
+func (c *reporter) PostMsgSend(resp interface{}, err error, duration time.Duration) {
+	if !c.opts.shouldLog(interceptors.FullMethod(c.service, c.method), err) {
+		return
+	}
+	// If the first message is logged skip the rest of the logging.
+	if c.firstMessageLogged {
+		return
+	}
+	c.firstMessageLogged = true
 
-func (c *reporter) PostMsgReceive(interface{}, error, time.Duration) {}
+	logger := c.logger.With(extractFields(tags.Extract(c.ctx))...)
+	logger = logger.With("grpc.recv.duration", duration.String())
+	if err != nil {
+		logger = logger.With("error", fmt.Sprintf("%v", err))
+	}
+
+	if c.isServer {
```

Comment:
hm I think for a "start", for server it's recv always, and for client it's send, so maybe we can put `isServer` above and return immediately if server?